pkgs/by-name/as/aseprite/package.nix

```diff
@@ -106,7 +124,7 @@ stdenv.mkDerivation rec {
         This version is not redistributable: https://dev.aseprite.org/2016/09/01/new-source-code-license/
         Consider supporting the developer: https://aseprite.org/#buy
       '';
-    maintainers = with maintainers; [ orivej ];
-    platforms = platforms.linux;
+    maintainers = with lib.maintainers; [ orivej vigress8 ];
```

Comment:
It is ideal to make a different commit for adding yourself as maintainer, the reason for that is that it can be reversed directly whenever you decide to no longer be a committer. And it won't affect the package.